dependencies.yaml

```diff
@@ -131,18 +137,21 @@ dependencies:
               - *libcusolver114
               - *libcusparse_dev114
               - *libcusparse114
-  doc:
+  docs_build:
     common:
-      - output_types: [conda, requirements]
-        packages:
-          - breathe
-      - output_types: [requirements]
-        packages:
-          - sphinx_markdown_tables
       - output_types: [conda]
         packages:
+          - breathe
           - doxygen>=1.8.20
-          - sphinx-markdown-tables
+          - ipython
+          - numpydoc
+          - pip
+          - pip:
+            - breathe
+            - pydata_sphinx_theme
+            - sphinx_copybutton
+            - sphinx_markdown_tables
+          - recommonmark
```

Comment:
`breathe` is on this list twice.

also, the `pip` method of installing packages should be reserved for packages that aren't available on Anaconda.org.

It looks like all of these `pip` packages are currently available on Anaconda.org:

- https://anaconda.org/conda-forge/pydata-sphinx-theme
- https://anaconda.org/conda-forge/breathe
- https://anaconda.org/conda-forge/sphinx-copybutton
- https://anaconda.org/conda-forge/sphinx-markdown-tables

Therefore this `pip` section should be removed and its dependencies should be moved up a level.